code/code/misc/parse.h

```diff
@@ -527,6 +527,7 @@ enum cmdTypeT {
      CMD_OPERATE,
      CMD_SPELLS,
      CMD_RITUALS,
+     CMD_PRAYERS,
```

Comment:
Injecting into the middle of enums is dangerous - you're shifting half of the values. Are you sure they're never stored anywhere, like in rent?

Reply:
Good call, resolved in https://github.com/sneezymud/sneezymud/pull/387/commits/2f07c8841d5d942d296a8d2e6b61f8b7d5a04a81